dataland-backend/backendOpenApi.json

```diff
@@ -3983,9 +3942,6 @@
           },
           "referencedReports": {
             "type": "object",
-            "additionalProperties": {
```

Comment:
No longer necessary?

Reply:
That is actually a good catch. I'm not sure myself whether this is as desired. At least, I have not intended to remove the CompanyReport reference here. Could be that this needs to be changed from devtools side - let me investigate this